pkgs/applications/window-managers/wio/default.nix

```
@@ -20,23 +20,15 @@
 
 stdenv.mkDerivation rec {
   pname = "wio";
-  version = "0.0.0+unstable=2021-06-01";
+  version = "0.0.0+date=2021-06-27";
```

How often do we want to come up with new version formats?

```suggestion
  version = "unstable-2021-06-27";
```